.travis.yml

```diff
@@ -8,6 +8,7 @@ before_install:
 - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.3.2
 - export PATH="$HOME/.yarn/bin:$PATH"
 - yarn global add greenkeeper-lockfile@1
+install: yarn --ignore-engines
```

Comment:
Why is this needed now?

Reply:
https://github.com/facebook/jest/blob/9cbc26bab656fa9e9952242b2d733a86c6a61d43/packages/jest/package.json#L13

Reply:
What about dropping Node.js 4.x support here, if Jest requires > Node.js 6.x?

Reply:
Mainly because we don't really need to, and this plugin isn't necessarily bound to a single version of Jest

Reply:
@ntwb Also, Jest supports Node 4 with custom environment targeting jsdom@9